masterlist.yaml

```
@@ -20680,6 +20750,101 @@ plugins:
+        condition: 'active("ClockworkSorting_Ing_CCCurios.esp")'
+      - <<: *includesX
+        subs: [ '"Additional Clockwork - CACO Ingredient Sorting Patch" (ClockworkSorting_Ing_CACO.esp)' ]
+        condition: 'active("ClockworkSorting_Ing_CACO.esp")'
+    clean:
+      - crc: 0x3297a568
+        util: "SSEEdit v4.0.3"
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087' ]
+  # Clockwork High poly Project Patches #
+  - name: 'ClockworkHPPCoalsFix.esp'
+    group: "Fixes & Resources"
+    msg:
+      - type: say
+        content: "Additional Clockwork's HPP Coals Fix is also installed. Consider using the merged plugin instead (included with installer)."
+        condition: 'active("ClockworkHPPHayFix.esp") and not active("ClockworkHPPMergedFixes.esp")'
+      - <<: *alreadyInX
+        subs: ["\"Additional Clockwork - High Poly Project Merged Patch\" (ClockworkHPPMergedFixes.esp)"]
+        condition: 'active("ClockworkHPPMergedFixes.esp")'
+      - <<: *requiresResources
+        subs: ["[Static Mesh Improvement Mod](https://www.nexusmods.com/skyrimspecialedition/mods/659)"]
+        condition: 'not file("textures/smim/clutter/charcoal/Charcoal01.dds") or not file("textures/smim/clutter/charcoal/Charcoal01_n.dds")'
+    clean:
+      - crc: 0xa4e097b9
+        util: "SSEEdit v4.0.3"
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087' ]
+  - name: 'ClockworkHPPHayFix.esp'
+    group: "Fixes & Resources"
+    msg:
+      - type: say
+        content: "Additional Clockwork's \"HPP Coals Fix\" (ClockworkHPPCoalsFix.esp) is also installed. Consider using the merged plugin instead."
+        condition: 'active("ClockworkHPPCoalsFix.esp") and not active("ClockworkHPPMergedFixes.esp")'
+      - <<: *alreadyInX
+        subs: ["\"Additional Clockwork - High Poly Project Merged Patch\" (ClockworkHPPMergedFixes.esp)"]
+        condition: 'active("ClockworkHPPMergedFixes.esp")'
+    clean:
+      - crc: 0xb00e3f81
+        util: "SSEEdit v4.0.3"
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087' ]
+  - name: 'ClockworkHPPMergedFixes.esp'
+    group: "Fixes & Resources"
+    msg:
+      - <<: *includesX
+        subs: ["\"Additional Clockwork - HPP Hay Fix\" (ClockworkHPPHayFix.esp)"]
+        condition: 'active("ClockworkHPPHayFix.esp")'
+      - <<: *includesX
+        subs: ["\"Additional Clockwork - HPP Coals Fix\" (ClockworkHPPCoalsFix.esp)"]
+        condition: 'active("ClockworkHPPHayFix.esp")'
+      - <<: *requiresResources
+        subs: ["[Static Mesh Improvement Mod](https://www.nexusmods.com/skyrimspecialedition/mods/659)"]
+        condition: 'not file("textures/smim/clutter/charcoal/Charcoal01.dds") or not file("textures/smim/clutter/charcoal/Charcoal01_n.dds")'
+    clean:
+      - crc: 0x4acde125
+        util: "SSEEdit v4.0.3"
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087' ]
```

Formatting changes + `*includesX` messages are redundant with above `*alreadyInX` messages:

```suggestion
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
    group: *fixesGroup
    msg:
      - <<: *requiresResources
        subs: [ '[Static Mesh Improvement Mod](https://www.nexusmods.com/skyrimspecialedition/mods/659/)' ]
        condition: 'not file("textures/smim/clutter/charcoal/Charcoal01.dds") or not file("textures/smim/clutter/charcoal/Charcoal01_n.dds")'
    clean:
      - crc: 0x4ACDE125
        util: 'SSEEdit v4.0.3'
```